samples/dnn/facial_landmarks_ie.cpp

```diff
@@ -0,0 +1,195 @@
+     "{ landmframework       | |     Optional name of an origin framework of the landmarkdetector. Detect it automatically if it does not set. }"
+     "{ facebackend          | 0 |   Choose one of computation backends for facedetector: "
+                                         "0: automatically (by default), "
+                                         "1: Halide language (http://halide-lang.org/), "
+                                         "2: Intel's Deep Learning Inference Engine (https://software.intel.com/openvino-toolkit), "
+                                         "3: OpenCV implementation }"
+     "{ landmbackend         | 0 |   Choose one of computation backends for facedetector: "
+                                         "0: automatically (by default), "
+                                         "1: Halide language (http://halide-lang.org/), "
+                                         "2: Intel's Deep Learning Inference Engine (https://software.intel.com/openvino-toolkit), "
+                                         "3: OpenCV implementation }"
+     "{ facetarget           | 0 |   Choose one of target computation devices for facedetector: "
+                                         "0: CPU target (by default), "
+                                         "1: OpenCL, "
+                                         "2: OpenCL fp16 (half-float precision), "
+                                         "3: VPU }"
+     "{ landmtarget          | 0 |   Choose one of target computation devices for landmarkdetector: "
+                                         "0: CPU target (by default), "
+                                         "1: OpenCL, "
+                                         "2: OpenCL fp16 (half-float precision), "
+                                         "3: VPU }"
+        );
+
+    parser.about("Use this script to run classification deep learning networks using OpenCV.");
+    if (argc == 1 || parser.has("help"))
+    {
+        parser.printMessage();
+        return 0;
+    }
+
+    //Parsing input arguments
+    const std::string facexmlPath = parser.get<std::string>("facestruct");
+    const std::string facebinPath = parser.get<std::string>("faceweights");
+    const Target faceTarget = parser.get<Target>("facetarget");
+    const Backend faceBackend = parser.get<Backend>("facebackend");
+
+    const std::string landmxmlPath = parser.get<std::string>("landmstruct");
+    const std::string landmbinPath = parser.get<std::string>("landmweights");
+    const Target landmTarget = parser.get<Target>("landmtarget");
+    const Backend landmBackend = parser.get<Backend>("landmbackend");
+
+    const Scalar color = Scalar(parser.get<int>("b"), parser.get<int>("g"), parser.get<int>("r"));
+
+    //Models' definition & initialization
+    Net faceNet = readNet(facexmlPath, facebinPath);
+    faceNet.setPreferableTarget(faceTarget);
+    faceNet.setPreferableBackend(faceBackend);
```

Comment:
There is only `DNN_INFERENCE_ENGINE_BACKEND` is supported fo IR models so you can emit `faceBackend` argument

Reply:
Have I got it right that it has no sense even to ask the user about backends as there is only one of them available, so I can delete parser keys too?